playbooks/byo/openshift-cluster/upgrades/docker/files/nuke_images.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Stop all running containers
+docker stop $(docker ps -q)
+
+# Delete all containers
+docker rm $(docker ps -a -q)
```

Comment:
This will leave container volumes behind, the correct line should be `docker rm -v ...`